common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/YubiKitNfcSmartcardCertBasedAuthManager.java

```diff
@@ -104,18 +104,29 @@ public void invoke(@NonNull NfcYubiKeyDevice value) {
     @Override
     void stopDiscovery(@NonNull Activity activity) {
         synchronized (sDeviceLock) {
-            if (isDeviceConnected()) {
+            mNfcDevice = null;
```

Comment:
We removed the check for isDeviceConnected?

Reply:
Are we assuming that disconnect() from this class will be called before this method everytime?

Reply:
I left it up to the developer to make sure that the user has disconnected their smartcard beforehand via the disconnect() method. I don't think anything adverse happens if something is still connected when we disable discovery, and adding this check in would mean we would need to include some sort of logic as a parameter to show the removal prompt dialog, because we need to tell the user to disconnect. 
However, I'm willing to add that logic back in, if I end up making the change mentioned earlier above regarding disconnect().